assembl/models/auth.py

```diff
@@ -759,6 +759,15 @@ def password_p(self):
     def password_p(self, password):
         from ..auth.password import hash_password
         if password:
+            # keep the current password to avoid the user to reuse it later
+            self.old_passwords.append(
+                OldPassword(password=self.password))
```

Comment:
Putting unsalted, unencrypted password in the db here. Old, but still sensitive data :(

Reply:
`self.password` is the hash & salted password here.

Reply:
Ahh of course. Okay, all good.

Reply:
I understood the same thing by reading the commit code outside of PR, I did not see that there were 2 different variables password and self.password. I think you should rename these 2 variables to make them really look different and more clear. This is a readability problem.